frontend-student/src/components/signup/Signup.jsx

```
@@ -0,0 +1,140 @@
+              onChange={signupFormik.handleChange}
+              value={signupFormik.values.address}
+            />
+          </div>
+          <button type="submit" className="w-full bg-blue-500 text-white py-2 rounded-md hover:bg-blue-600">
+            Sign Up
+          </button>
+        </form>
+        <div className="text-center mt-4">
+          <p className="text-gray-700">
+            Already have an account?{" "}
+            <button className="text-blue-500 hover:underline" onClick={() => navigate("/login")}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)